integration-tests/invoke_test.go

```diff
@@ -55,17 +55,22 @@ func TestInvokes(t *testing.T) {
 		testAzureInvokeVM(t, tmpConfigFile, *workingDir)
 	})
 
+	t.Run("az.create-aks", func(t *testing.T) {
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
We have limited aks resources and it takes usually very long to create an aks cluster, similarly to eks. I would rather have it as a manual test